### PR TITLE
Change company address 

### PR DESCRIPTION
**Pull request by freevova**
Change the address to:
Address 401 E 1st St Suite 1868 - 0039, Sanford FL , 32772

Change the map as well.

### CODE REVIEW

**Comment by freevova** [comment]
@EnragedWildkin Changed address but when I press on map image it shows me old adress. How can I correct this bug?

File: contact.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <div class="contacts_wrap">
       <div class="map">
         <a href="https://goo.gl/maps/MKsk2" target="_blank" rel="nofollow" title="Open on google map in new tab">
```
Comment (mir4a) [inline contact.html]: @freevova change this link

**Comment by EnragedWildkin** [comment]
@freevova Link in view https://github.com/jetthoughts/jetthoughts.github.io/blob/master/contact.html#L14

**Comment by freevova** [comment]
@EnragedWildkin Please review.

File: _includes/sections/footer_contacts.html [hunk 2]
```diff
@@ -9,8 +9,9 @@ <h1 class="heading_one">Get In Touch</h1>
       </div>
 
       <div class="address_wrap">
-        2360 Corporate Circle STE 400 <br>
-        Henderson 89074-7722 NV, USA
+        401 E 1st St Suite 1868 - 0039 <br>
+        Sanford FL , 32772
```
Comment (EnragedWildkin) [inline _includes/sections/footer_contacts.html]: Please remove whitespace before `,`

**Pull request merged**